### DISCUSSION TITLE
The description of the doctor's talents is displayed incorrectly

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13839] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The description of the doctor's talents is displayed incorrectly

![2024-04-26_02-41-52](https://github.com/FakeFishGames/Barotrauma/assets/47295383/bcac3b0d-57eb-4f70-afdb-dbd76a506d56)

### Reproduction steps

Just look at the doctor's talents description

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13813
